setup_env.sh

```diff
@@ -0,0 +1,9 @@
+#!bin/bash
+
+AWS_REGION=$(cat ~/.aws/config | grep region | awk -F "=" '{print $NF}' | sed -e 's/^ *//')
```

Comment:
Let's use single command here, e.g. `awk -F "[ =]+" '/region/{print $2; exit}' ~/.aws/config`
Also, it could be extracted to be a function.